src/python_be.cc

```diff
@@ -2128,6 +1451,40 @@ ModelState::ModelState(TRITONBACKEND_Model* triton_model)
   }
 }
 
+TRITONSERVER_Error*
+ModelState::LaunchAutoCompleteStubProcess()
+{
+  Stub() = std::make_unique<StubLauncher>("AUTOCOMPLETE_STUB");
+  RETURN_IF_ERROR(Stub()->Initialize(this));
+  try {
+    RETURN_IF_ERROR(Stub()->Launch());
+  }
+  catch (const BackendModelException& ex) {
+    Stub()->TerminateStub();
+    // Need to make sure that the shared memory pool is destructed properly
```

Comment:
Can you expand on this comment? If we destruct the stub, Would this automatically delete the stub launcher as well?

Reply:
I found that when destructing the stub here, `shm_pool` would not be reset correctly and would cause shm page not cleaned if we don't reset it here.

Reply:
What if we reset the stub here (i.e. call Stub().reset())?